src/components/MyAccount/CheckoutsTab/RenewButton.tsx

```diff
@@ -123,10 +136,21 @@ const RenewButton = ({
         buttonType="secondary"
         id={`renew-${checkout.id}`}
         onClick={handleClick}
-        aria-disabled={isButtonDisabled}
+        aria-disabled={isButtonDisabled || showLoadingState}
         aria-label={`Renew ${checkout.title}`}
       >
-        Renew
+        {showLoadingState && (
+          <ProgressIndicator
+            id={"renew-loading"}
+            labelText="Renew"
```

Comment:
This only renders in the loading state so it should be "Loading" instead of "Renew".